### ISSUE TITLE
Add documentation for config of tracing

### ISSUE THREAD

**Issue by tomas-langer** [oracle/helidon#3345]
Helidon MP 

We currently lack documentation on how to configure tracing in MP.

Example below - to disable all built-in tracing in Helidon MP:
```proprerties
# this disables Helidon tracing for web server and security
tracing.components.web-server.enabled=false
tracing.components.security.enabled=false

# This disables MP Tracing required by spec
mp.opentracing.server.skip-pattern=.*
```

**Comment by tomas-langer** [comment]
Probably also add the table from SE tracing docs that shows all possible components, spans and logs we use + add details on how to configure using MP config.

**Comment by dalexandrov** [comment]
Fixed with #3368
